gptqmodel/quantization/quantizer.py

```diff
@@ -30,9 +30,11 @@ def quantize(x, scale, zero, maxq, requires_groupwise_processing: bool):
     if maxq < 0:
         return (x > scale / 2).float() * scale + (x < zero / 2).float() * zero
     if requires_groupwise_processing:
+        scale = torch.where(scale == 0, torch.tensor(1e-8, device=scale.device), scale)
```

Comment:
Just some small change requests:

1. Move the scale mod to outside of the if/else.
2. Wrap with check before mod and log so we know this module/model has problems and not just totally ignore this issue.:

```py
if (scale == 0).any():
   scale = torch.where(scale == 0, torch.tensor(1e-8, device=scale.device), scale)
   log.error(f"Quantize: zero scale found and bypassed. This module may not be properly trained or received enough calibration data causing unmanageable outliers. If MoE module, please check with model maker and/or feed with larger calibration data. : `{scale}`. ")

Reply:
@ywlq  I don't see any code changes? Please make the small the changes above and push.